server/src/internalClusterTest/java/org/elasticsearch/cluster/allocation/AwarenessAllocationIT.java

```diff
@@ -307,4 +309,22 @@ public void testAwarenessZonesIncrementalNodes() {
         assertThat(counts.get(B_1), equalTo(2));
         assertThat(counts.get(noZoneNode), equalTo(2));
     }
+
+    public void testForceAwarenessSettingValidation() {
+        final String prefix = AwarenessAllocationDecider.CLUSTER_ROUTING_ALLOCATION_AWARENESS_FORCE_GROUP_SETTING.getKey();
+
+        expectThrows(SettingsException.class, () ->
```

Comment:
I could be wrong, but I think this results in a 500 internal error at the REST layer? If so, I think we should fix that to be a 400 in the validation of the forced awareness setting.

Reply:
Yes I think you're right. Wrapped in an IAE in 6977ba5, although maybe `SettingsException` should always be a 400. That's a debate for another day...